packages/workspace/src/core/project-graph/build-project-graph.ts

```diff
@@ -82,6 +82,7 @@ export async function buildProjectGraphUsingProjectFileMap(
   let cachedFileData = {};
   if (
     cache &&
+    cache.version === '5.0' &&
```

Comment:
Shouldn't this be a constant somewhere? Like `LATEST_PROJECT_GRAPH_VERSION`?

I imagine there are at least a couple more references to `"5.0"`

Reply:
Good point

Reply:
That is true, we should have a `const`. We can do it as a separate change though.